File: Pattern 8/Pattern 8.cpp
class Solution{
public:
	
	void printTriangle(int n) {
	    int x=n;
        for(int r=1;r<=n;r++){
            for(int sp=1;sp<=r-1;sp++)
                cout<<" ";
            for(int st=1;st<=2*x-1;st++)
                cout<<"*";
           x=x-1;
           cout<<"\n";
        }
	}
};

File: Pattern 8/Pattern 8.py
class Solution:
    def printTriangle(self, N):
        for i in range(N):
            for j in range(i):
                print(" ",end ="")
                
            for j in range(2*N - (2*i+1), 0, -1):
                print("*",end ="")
            print()

